src/Sentry.Unity/Default/UnknownPlatform.cs

```diff
@@ -11,14 +11,23 @@ public static class SentryUnknownPlatform
     {
         public static void Configure(SentryUnityOptions options)
         {
+            options.DiagnosticLogger?.Log(SentryLevel.Debug,
+                "Configuring on an unknown platform - some options may be overridden to improve compatibility.");
```

Comment:
```suggestion
                "Configuring on an unsupported platform - some options may be overridden to improve compatibility.");
```
What do you think about unsupported instead of unknown? Because we actually do know the current platform, it's just not supported (yet).

Reply:
I was considering this. Why I chose not to is because it's not really "unsupported", just not "officially supported"